### ISSUE TITLE
Improve calendar modal accessibility

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1537]
Calendar modals are not accessible, meaning that they are either not focusable or the focus can leave the modal while it's still open.

When a modal is open, it should allow users to interact with elements within the modal using the keyboard navigation, and the focus shouldn't leave the modal
